docs/pages/documentation/6-developing-with-java/development-overview.md

```diff
@@ -22,13 +22,13 @@ All Grakn applications require the following Maven dependency:
 <dependencies>
   <dependency>
     <groupId>ai.grakn</groupId>
-    <artifactId>grakn-graph</artifactId>
+    <artifactId>grakn-kb</artifactId>
```

Comment:
Nope, but same deal it will be a while before these docs get released so any changes I am preemptively making the changes now before we forget.